initrd/.ash_history

```diff
@@ -15,3 +15,6 @@ seal-totp
 #Verify Intel ME state:
 cbmem --console | grep '^ME'
 cbmem --console | less
+# Reboot/power off (important for devices with no keyboard to escape recovery shell)
+reboot
```

Comment:
I would suggest adding comment after comment so that user can see those+ comments when going up and down where comment would be neglicted at call.

Reply:
Added that, thanks, good suggestion :bow: